src/Cardano/Wallet/CoinSelection/Fee.hs

```diff
@@ -341,3 +350,62 @@ computeFee (CoinSelection inps outs chgs) =
     -- by definition, impossible; unless we messed up real hard.
     collapse [] _ =
         invariant "outputs are bigger than inputs" (undefined) (const False)
+
+-- | A linear equation on the transaction size. Represents the @\s -> a + b*s@
+-- function where @s@ is the transaction size in bytes, @a@ and @b@ are
+-- constant coefficients.
+data TxSizeLinear = TxSizeLinear Double Double
```

Comment:
To make things slightly more explicit:

`Double` -> `Quantity "byte" Double` 
`Double` -> `Quantity "lovelace/byte" Double`

Reply:
will do it in the next commit

Reply:
done